src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -490,7 +490,7 @@ public static function get_user_data() {
 				 *
 				 * @param WP_Theme_JSON_Data Class to access and update the underlying data.
 				 */
-				$theme_json = apply_filters( 'theme_json_user', new WP_Theme_JSON_Data( $config, 'custom' ) );
+				$theme_json = apply_filters( 'theme_json_user_data', new WP_Theme_JSON_Data( $config, 'custom' ) );
```

Comment:
A bit unrelated, but it's odd that `custom` is used here which isn't documented in https://github.com/wordpress/wordpress-develop/blob/711b74c21bef856677fd97d82d17f7761d0d602a/src/wp-includes/class-wp-theme-json-data.php#L40.